networks/fabric/v2/v2.1.0/2org1peergoleveldb_raft/api/fabric-api-solo-go-prometheus.yaml

```diff
@@ -74,7 +74,7 @@ channels:
             - id: fixed-asset
```

Comment:
Chaincode metadataPath needs to be changed to `src/fabric/api/fixed-asset-base/metadata`